.github/workflows/node.js.yml

```diff
@@ -71,6 +71,10 @@ jobs:
       run: |
         npm install
 
+    - name: Install Playwright dependencies
+      run: npx playwright install-deps
```

Comment:
I think I should change that to `playwright-webkit` so that it uses a local version.